src/software/thunderscope/binary_context_managers.py

```diff
@@ -479,8 +487,8 @@ def send_ci_input(
         self,
         gc_command: proto.ssl_gc_state_pb2.Command,
         team: proto.ssl_gc_common_pb2.Team,
-        final_ball_placement_point=None,
-    ):
+        final_ball_placement_point: Any = None,
```

Comment:
Looking here:

https://github.com/UBC-Thunderbots/Software/blob/83d226fe1e56d0ab93991a0acdae88a344ce0ec5/src/software/ai/hl/stp/play/ball_placement_play_test.py#L20

it looks like the type of the point is `tbots_cpp.Point` from `import software.python_bindings as tbots_cpp` (I think the import is already added to this file on the latest commit on master)